style.css

```diff
@@ -0,0 +1,352 @@
+  background-color: white;
+  background-size: 100%;
+  border: 2px solid white;
+  border-radius: 0 0 0 50px;
+}
+
+.list {
+  display: none;
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: white;
+  padding-bottom: 1rem;
+  z-index: 1;
+}
+
+.explain {
+  width: 100%;
+  padding-top: 4rem;
+}
+
+.logo {
+  margin: 1rem 0 0 2rem;
+  text-decoration: none;
+  font-size: 25px;
+  color: #6070ff;
+}
+
+.union {
+  margin: 1rem 3rem 0 0;
+}
+
+h1 {
+  font-size: 40px;
+  margin: 5rem 0 0 2rem;
+}
+
+.header-text {
+  color: #344563;
+  font-size: 16px;
+  margin: 1rem 0 0 2rem;
+  width: 70%;
+  line-height: 2.5rem;
+}
+
+.about a {
+  text-decoration: none;
+  color: #7f8cff;
+  font-size: 18px;
+  font-weight: 600;
+}
+
+.about {
+  margin: 2rem 0 0 2rem;
+}
+
+.image ul {
+  display: flex;
+  list-style: none;
+  margin: 1.5rem 0 6rem 2rem;
+}
+
+.list1 li {
+  margin: 1rem 2rem 0 0;
+}
+
+.list2 li {
+  padding: 4px 12px;
+  margin: 1rem 0.6rem 0 0;
+  border: 2px solid #dadbff;
+  background-color: #ebebff;
+  border-radius: 0.5rem;
+  list-style: none;
+  font-size: 12px;
+  font-weight: bolder;
+  color: #6070ff;
+}
+
+.list3 li {
+  padding: 4px 12px;
+  margin: 1rem 0.6rem 0 0;
+  border: 0.1px solid #dadbff;
+  background-color: #ebebff;
+  border-radius: 0.5rem;
+  list-style: none;
+  font-size: 12px;
+  font-weight: bolder;
+  color: #6070ff;
+}
+
+.let-connect a {
+  text-decoration: none;
+  color: #7f8cff;
+  font-size: 18px;
+  font-weight: 600;
+}
+
+.languages li {
+  list-style: none;
+}
+
+.frame-right li,
+.last-icons li,
+.skill-right li {
+  list-style: none;
+}
+
+.image ul li {
+  margin-right: 2rem;
+  font-size: 2rem;
+  color: #505f79;
+}
+
+.image ul li a {
+  color: #505f79;
+}
+
+.sectiontemplate {
+  display: block;
+  margin: 4rem auto auto auto;
+  width: 90%;
+  background-color: white;
+  border-radius: 50px;
+}
+
+.img1 {
+  display: block;
+  margin: auto auto 0 auto;
+  width: 90%;
+  padding-top: 2rem;
+}
+
+.list1 {
+  display: flex;
+}
+
+.list2 {
+  display: flex;
+}
+
+.list3 {
+  display: flex;
+}
+
+.Section2Text {
+  padding: 6rem 0 3rem 3rem;
+}
+
+.special {
+  color: #7a869a;
+}
+
+.can {
+  list-style: none;
+}
+
+p {
+  color: #344563;
+  margin-top: 1rem;
+  line-height: 2rem;
+  font-size: 19px;
+  width: 80%;
+}
+
+.btn {
+  margin-top: 1rem;
+  padding: 0.8rem;
+  border: 1px solid #6070ff;
+  border-radius: 0.5rem;
+  font-size: 16px;
+  font-weight: bolder;
+  color: #6070ff;
+  background-color: #fff;
+}
+
+.btn:hover {
+  background-color: #7f8cff;
+  color: white;
+}
+
+.btn:active {
+  background-color: darkblue;
+  color: white;
+}
+
+.btn:disabled {
+  background-color: white;
+  color: black;
+}
+
+.Section3Text {
+  padding: 6rem 0 3rem 3rem;
+}
+
+.SectionText {
+  padding: 3rem 0 3rem 3rem;
+}
+
+h2 {
+  color: #172b4d;
+  font-size: 35px;
+  margin: -2rem 0 1rem 0;
+}
```

Comment:
- [ ] [Optional]

Kindly use CSS classes for styling the HTML elements As CSS classes are reusable. 👍🏻